types/facebook-js-sdk/index.d.ts

```diff
@@ -1,20 +1,65 @@
-// Type definitions for the Facebook Javascript SDK 3.2
+// Type definitions for the Facebook Javascript SDK 3.3
 // Project: https://developers.facebook.com/docs/javascript
 // Definitions by:  Amrit Kahlon    <https://github.com/amritk>
 //                  Mahmoud Zohdi   <https://github.com/mahmoudzohdi>
 //                  Marc Knaup      <https://github.com/fluidsonic>
+//                  Ben Grynhaus    <https://github.com/bengry>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
 
 import fb = facebook;
 declare var FB: fb.FacebookStatic;
 declare namespace facebook {
-
+    type FacebookEventType =
+        | 'auth.authResponseChange'
+        | 'auth.logout'
+        | 'auth.login'
+        | 'auth.statusChange'
+        | 'xfbml.render';
+
+    type ResponseStatus =
```

Comment:
This is a misleading name.
I'd suggest `LoginStatus` in sync with [FB.getLoginStatus()](https://developers.facebook.com/docs/reference/javascript/FB.getLoginStatus/)

Reply:
Agreed. Changed.